### DISCUSSION TITLE
Using In built client breaking news to announce new things or updates about Chromiecraft

### DISCUSSION THREAD

**Discussion by HasieEST** [chromiecraft/chromiecraft#4797] [Suggestions]
While you do use Discord and Website to announce stuff, it wouldn't hurt to use already existing breaking news section on client login screen. Think Ascension and Warmane servers use that feature to communicate with the players. Adds another avenue to share info, thus  increasing chance that some people will know about new stuff, without need to visit website or discord.

**Comment by heyitsbench** [comment]
Currently there are three (really only two) options for this: one is to distribute a patch for players to modify the URL for the actual breaking news that Blizzard used and point it to one specific for use with ChromieCraft/AzerothCore happenings.

Another is to try to do what Warmane does which has only just recently been touched whatsoever by AC members at all, and still hasn't been fleshed out and likely won't be added to the base AC core, and as such won't be seeing the light of day with CC.

Option three of course is to do neither and keep on keeping on.
